docs/docs/cli/reference/README.md

```diff
@@ -1,6 +1,8 @@
 # CLI Reference
```

Comment:
I added docs for `pd publish` and `pd dev`, and updated all docs to reference the terms components, sources or actions, where relevant.